silkrpc/core/rawdb/util.hpp

```diff
@@ -0,0 +1,29 @@
+/*
+   Copyright 2020 The Silkrpc Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#ifndef SILKRPC_CORE_RAWDB_UTIL_HPP_
+#define SILKRPC_CORE_RAWDB_UTIL_HPP_
+
+#include <string>
+
+#include <silkrpc/common/util.hpp>
+
+namespace silkrpc {
+
+silkworm::Bytes generate_composity_storage_key(const evmc::address& address, uint64_t incarnation, const uint8_t (&hash)[silkworm::kHashLength]);
```

Comment:
Rename this function as `composite_storage_key` (please note `composite`, not `composity`)

Reply:
ok